sdk/src/types/block/error.rs

```diff
@@ -211,7 +211,7 @@ impl fmt::Display for Error {
             }
             Self::InvalidAddress => write!(f, "invalid address provided"),
             Self::InvalidAddressKind(k) => write!(f, "invalid address kind: {k}"),
-            Self::InvalidAccountIndex(index) => write!(f, "invalid account index: {index}"),
+            Self::InvalidWalletIndex(index) => write!(f, "invalid account index: {index}"),
```

Comment:
Undo this change, account was correct here

Reply:
oopsie

Reply:
undone